commands/cli.py

```diff
@@ -0,0 +1,35 @@
+from typing import Any
+
+from cleo.application import Application
+
+import commands
+from starlite import Starlite
+
+
+class Cli:
+    """CLI Add CLI commands using cleo https://github.com/python-poetry/cleo .
+    New commands should be placed in the commands folder and added in
+    __init__.py.
+
+    usage example:
+        # ...
+        app = Starlite(route_handlers=[hello_world])
+        if __name__ == '__main__':
+            from commands.cli import Cli
+            cli = Cli(app)
+            cli.run()
+    """
+
+    def __init__(self, starletapp: Starlite):
+        self.st_app = starletapp
+        self.cli = Application()
+        for command in commands.__all__:
+            cli_command = getattr(commands, command)
+            try:
+                self.cli.add(cli_command(self.st_app))
+            except Exception:
+                self.cli.add(cli_command())  # pylint: disable = no-value-for-parameter
+
+    def run(self) -> int | Any:
```

Comment:
int | Any is a weak typing. What does cli.run return?

Reply:
it should be an `int` (afaik), but mypy does not like that. `error: Returning Any from function declared to return "int"  [no-any-return]`  (??)

However, it does accept this:

    def run(self) -> int:
        errorcode: int = self.cli_.run()
        return errorcode 

I think that should not be necessary. However, in this case it does make it a bit more readable.

Reply:
So your issue is `cleo` has not been added to the `additional_dependencies` of MyPy and the other tools in `.precommit-config.yaml`. This You don't get proper type checks